### PR TITLE
Fix #2681: Remove onboarding_java_proto_lite export

### LINKED ISSUE #2681: Remove :onboarding_java_proto_lite export from the model library
This task is part of the build system migration from Gradle to Bazel. A prerequesite before working on this issue is that you are able to build Oppia using Bazel. Check out [Oppia Bazel guide](https://github.com/oppia/oppia-android/wiki/Oppia-Bazel-Guide) for Bazel terminology and more on what Bazel does.

## 1. Remove the export

In 

`/oppia-android/model/BUILD.bazel`

Remove :onboarding_java_proto_lite from the list of exports in the model library

```BUILD
android_library(
    name = "model",
    visibility = ["//visibility:public"],
    exports = [
        ":event_logger_java_proto_lite",
        ":exploration_java_proto_lite",
        ":interaction_object_java_proto_lite",
        ":onboarding_java_proto_lite",
        ":profile_java_proto_lite",
        ":question_java_proto_lite",
        ":subtitled_html_java_proto_lite",
        ":subtitled_unicode_java_proto_lite",
        ":thumbnail_java_proto_lite",
        ":topic_java_proto_lite",
        ":translation_java_proto_lite",
        ":voiceover_java_proto_lite",
    ],
)
```

## 2. Add public visibility to the java_lite_proto_library

Add 

`    visibility = ["//visibility:public"],`

to the java_lite_proto_library that has the name onboarding_java_proto_lite.

## 3. Build the app using Bazel

Libraries that are indirectly depending on `onboarding_java_proto_lite` will now fail to build.  Try to build the app with Bazel and add the deps dependency

`//model:onboarding_java_proto_lite,`

to all the libraries that are now failing to build. You will have to look around for BUILD.bazel files that have libraries that require this replaced dependency.

These libraries will already have the //model dependency and since we removed the :onboarding_java_proto_lite from the model library itself, they will need to add the newly available public library directly as a dependency. (Not all libraries with the //model dependency may require the newly available library)

Once the app builds, you're done!

### PR DESCRIPTION
**Pull request by Sparsh1212**
## Explanation

Fixes #2681: Remove onboarding_java_proto_lite export from the model library.
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by Sparsh1212** [comment]
@fsharpasharp @rt4914  PTAL.

**Comment by Sparsh1212** [comment]
@BenHenning Can you please update the branch here also so as to retrigger the build just like in the case of [#2699 ](https://github.com/oppia/oppia-android/pull/2699)

File: model/BUILD.bazel [hunk 1]
```diff
@@ -67,6 +67,7 @@ proto_library(
 java_lite_proto_library(
     name = "onboarding_java_proto_lite",
     deps = [":onboarding_proto"],
+    visibility = ["//visibility:public"],
```
Comment (anandwana001) [inline model/BUILD.bazel:79]: Bazel lint error, Fix like this.  

```
java_lite_proto_library(
    name = "onboarding_java_proto_lite",
    visibility = ["//visibility:public"],
    deps = [":onboarding_proto"],
)
```

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Bazel lint fix.

File: model/BUILD.bazel [hunk 2]
```diff
@@ -66,7 +66,8 @@ proto_library(
 
 java_lite_proto_library(
     name = "onboarding_java_proto_lite",
-    deps = [":onboarding_proto"],
+    visibility = ["//visibility:public"],
+    deps = [":onboarding_proto"],
```
Comment (anandwana001) [inline model/BUILD.bazel]: there is whitespace at the end of this failing with causing the failure

Screenshot 2021-02-14 at 09 52 46

**Review by anandwana001 — CHANGES_REQUESTED** [review]

**Comment by Sparsh1212** [comment]
@anandwana001 I have made the lint fixes and the build checks have also passed successfully. PTAL

**Review by fsharpasharp — APPROVED** [review]

**Review by anandwana001 — APPROVED** [review]
Bazel Lint fix LGTM

**Review by BenHenning — APPROVED** [review]
Thanks @Sparsh1212! LGTM.

**Pull request merged**